apidoc/Titanium/UI/ListView.yml

```diff
@@ -711,6 +711,13 @@ properties:
     since: "5.4.0"
     platforms: [iphone, ipad]
 
+  - name: AllowsMultipleSelectionDuringEditing
+    summary: Determines whether more than one row can be selected while editing the table.
+    type: Boolean
+    default: false
```

Comment:
Did you implement the getter? Otherwise, this will return undefined and should be `default: undefined (behaves as false)`.